packages/@expo/cli/src/login/index.ts

```diff
@@ -27,6 +29,7 @@ export const expoLogin: Command = async (argv) => {
         `-u, --username <string>  Username`,
         `-p, --password <string>  Password`,
         `--otp <string>           One-time password from your 2FA device`,
+        // `-s, --sso                Log in with SSO`,
```

Comment:
There should be a valid message in help, no?

Reply:
We're hiding it until it SSO is released